js/src/components/google-ads-account-card/connected.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
💅 / ❓  Why in `js/src/components/wpcom-account-card/index.js` this is named `connected-wpcom-account-card.js` whereas here is just `connected.js`?

Reply:
I originally intended to minimize the number of renaming files, so I used their original names. Considering that there are two exposed components, which may not match when globally searching for the codebase, I changed their internal names to the same one by https://github.com/woocommerce/google-listings-and-ads/pull/1374/commits/c0f7fe8fbfc6a681816e8af945c49fab6514aede.